powertools-e2e-tests/pom.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>powertools-parent</artifactId>
+        <groupId>software.amazon.lambda</groupId>
+        <version>1.12.3</version>
+    </parent>
+
+    <artifactId>powertools-e2e-tests</artifactId>
+    <description>AWS Lambda Powertools for Java End-To-End Tests</description>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Is forcing the compiler and language version intentional, or should it flow down from the parent like the rest of the modules?

Reply:
It is intentional, we don't really care not to be compliant with java 8 (it won't be deployed on AWS).